docs/en/sql-reference/window-functions/first_value.md

```diff
@@ -0,0 +1,79 @@
+---
+slug: /en/sql-reference/window-functions/first_value
+sidebar_label: first_value
+sidebar_position: 3
+---
+
+# first_value
+
+Returns the first value evaluated within its ordered frame. By default, NULL arguments are skipped, however the `RESPECT NULLS` modifier can be used to override this behaviour.
+
+**Syntax**
+
+```sql
+first_value (column_name) [RESPECT NULLS]
```

Comment:
```suggestion
first_value (column_name) [[RESPECT NULLS] | [IGNORE NULLS]]
```

You can explicitly declare it as `IGNORE NULLS` too. It doesn't do anything since it's the default, but it's there for compatibility.